api/tests/functional-tests/backend/core/test_evaluation.py

```diff
@@ -421,13 +421,47 @@ def test_get_evaluations(
     assert evaluations_by_dataset_model_eval_id[0].id == created_2[0].id
 
     # make sure stratifying works by dataset and model
-    evaluations_by_dataset_model_eval_id = core.get_evaluations(
+    evaluations_by_dataset_model_eval_id, _ = core.get_evaluations(
         db=db,
         dataset_names=[finalized_dataset],
         model_names=[finalized_model],
     )
     assert len(evaluations_by_dataset_model_eval_id) == 2
 
+    # test pagination
+    with pytest.raises(ValueError):
+        # offset is greater than the number of items returned in query
+        evaluations, headers = core.get_evaluations(
+            db=db,
+            dataset_names=[finalized_dataset],
+            model_names=[finalized_model],
+            offset=6,
+            limit=1,
+        )
+
+    evaluations, headers = core.get_evaluations(
+        db=db,
+        dataset_names=[finalized_dataset],
+        model_names=[finalized_model],
+        offset=1,
+        limit=1,
+    )
+
+    assert len(evaluations) == 1
```

Comment:
Can you add a value check to show that the output is deterministic?

Reply:
none of the tests here check values because there isn't really anything deterministic to check. here's an example output from this breakpoint:

```python
# print(evaluations)

[EvaluationResponse(id=188, model_name='test_model', datum_filter=Filter(dataset_names=['test_dataset'], dataset_metadata=None, model_names=None, model_metadata=None, datum_uids=None, datum_metadata=None, task_types=None, annotation_metadata=None, require_bounding_box=None, bounding_box_area=None, require_polygon=None, polygon_area=None, require_raster=None, raster_area=None, labels=None, label_ids=None, label_keys=None, label_scores=None), parameters=EvaluationParameters(task_type=<TaskType.CLASSIFICATION: 'classification'>, convert_annotations_to_type=None, iou_thresholds_to_compute=None, iou_thresholds_to_return=None, label_map=None, recall_score_threshold=0.0, compute_pr_curves=None, pr_curve_iou_threshold=0.5), status=<EvaluationStatus.PENDING: 'pending'>, created_at=datetime.datetime(2024, 4, 12, 15, 30, 18, 963293, tzinfo=datetime.timezone.utc), metrics=[], confusion_matrices=[], ignored_pred_labels=None, missing_pred_labels=None)]
```

the id changes each run. let me know what you'd want to assert from the above